KabuKit/Classes/core/Swift/TransitionRequest.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+open class TransitionRequest<ContextType, ExpectedResultType> {
```

Comment:
Why `Expected` ?

Reply:
The `Scene` that is received `TransitionRequest` does not necessarily reply anything.
_`Scene` can reply `nil`._
So, I named this generic type to ExpectedResultType.

But,  I think about the fact that `Scene can reply nil` is already expressed by Optional Type, now.
So, I rename `ExpectedResultType`  to `ReturnValue`
_ and rename `ContextType` to `Context`. (same to Scene)_